src/js/webrequest.js

```diff
@@ -103,6 +103,7 @@ function onBeforeRequest(details){
       };
       chrome.tabs.sendMessage(tab_id, msg);
 
+      badger.checkForDNTPolicy(requestDomain, badger.storage.getNextUpdateForDomain(requestDomain));
```

Comment:
Could we make both of these calls in `webrequest.js` async, to avoid delaying request resolution as much as we can?   Done elsewhere in `heuristicblocking.js`:
https://github.com/EFForg/privacybadger/blob/1e3d0e748b1027b5854d06afb00237702d65bef8/src/js/heuristicblocking.js#L124-L126